proof/invariant-abstract/Ipc_AI.thy

```diff
@@ -446,7 +446,7 @@ end
 
 (* FIXME: can some of these assumptions be proved with lifting lemmas? *)
 locale Ipc_AI_2 = Ipc_AI state_ext_t
-  for state_ext_t :: "'state_ext::state_ext itself"
+  for state_ext_t :: "'state_ext::state_ext itself" +
```

Comment:
strange. was this a broken outcome of the merge?

Reply:
Yep, I made a mistake resolving a conflict and fixing it here was easier than amending the merge commit to fix it there.